migration-tool/src/main/java/software/amazon/awssdk/migration/internal/recipe/NumberToDuration.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package software.amazon.awssdk.migration.internal.recipe;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.openrewrite.ExecutionContext;
+import org.openrewrite.Option;
+import org.openrewrite.Recipe;
+import org.openrewrite.TreeVisitor;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.JavaTemplate;
+import org.openrewrite.java.MethodMatcher;
+import org.openrewrite.java.tree.J;
+import software.amazon.awssdk.annotations.SdkInternalApi;
+
+/**
+ * Convert the method parameter from numeric types to duration.
+ */
+@SdkInternalApi
+public class NumberToDuration extends Recipe {
+    @Option(displayName = "Method pattern",
+        description = "A method pattern that is used to find matching method invocations.",
+        example = "com.amazonaws.ClientConfiguration setRequestTimeout(int)")
+    private final String methodPattern;
+
+    @Option(displayName = "Time Unit",
+        description = "The TimeUnit enum value to convert. Defaults to `MILLISECONDS`.",
+        example = "MILLISECONDS",
+        required = false)
+    private final TimeUnit timeUnit;
+
+    @JsonCreator
```

Comment:
I cant remember, why we need to use Jackson's annotations?

Reply:
OpenRewrite requires it to initialize the recipe. 
This error will be thrown if we move it.
```
java.io.UncheckedIOException: com.fasterxml.jackson.databind.exc.InvalidDefinitionException: 
Cannot construct instance of `software.amazon.awssdk.migration.internal.recipe.NumberToDuration` 
(no Creators, like default constructor, exist): cannot deserialize from Object value 
(no delegate- or property-based Creator)
 at [Source: (byte[])[232 bytes]; byte offset: #75]
```